packages/strapi-design-system/src/Divider/Divider.js

```diff
@@ -1,11 +1,16 @@
+import React from 'react';
 import PropTypes from 'prop-types';
 import { Box } from '../Box';
 import styled from 'styled-components';
 
-export const Divider = styled(Box)`
+const DividerWrapper = styled(Box)`
   height: 1px;
+  margin: 0;
+  border: none;
 `;
 
+export const Divider = (props) => <DividerWrapper {...props} as="hr" />;
```

Comment:
WDYT of the following:

```

const Divider = styled(Box)`
  height: 1px; // should we use rem?
  margin: 0;
  border: none;
`;

Divider.defaultProps = {
as: 'hr'
}

```

Reply:
That means we will be able to change the `as` props. Do we want that? IMO we should force the prop

Reply:
Can't we force it using the `.withConfig` ?

Reply:
What is the easiest to read and to reason about for people?

Reply:
We also can do `styled(() => <Box as="hr" \>)`. So that, we don't need to create a new component.

Reply:
```jsx
() => <Box as="hr" \> 
```

is the new (anonymous) component 😊

Reply:
yes a new comp will be created anyway haha